public/resources/js/form.js

```diff
@@ -1,76 +1,89 @@
+        pass.style.backgroundColor = "rgba(236, 240, 22, 0.486)";
+        msg.style.color = "#f3eb51";
+        msg.style.marginBottom = "10px";
+        msg.style.top = "-8px";
+        msg.style.position = "relative";
+    } else if(pass.value.length >= 8){
+        str.innerHTML = "strong";
+        pass.style.backgroundColor = "rgba(81, 182, 14, 0.651)";
+        msg.style.color = "#5bec64";
+        msg.style.marginBottom = "10px";
+        msg.style.top = "-8px";
+        msg.style.position = "relative";
+    }
+    checkPasswordConfirmation();
+});
 
-    copass.addEventListener('input',() =>{
-        checkPasswordConfirmation();
-    });
+copass.addEventListener('input',() =>{
+    checkPasswordConfirmation();
+});
 
-    email.addEventListener('input',() =>{
-        validateEmail();
-    });
-    
-    function checkPasswordConfirmation() {
-        if(copass.value.length > 0){
-            nmsg.style.display = "block";
-        }else{
-            nmsg.style.display = "none";
-            return;
-        }
-        if(copass.value === pass.value ){
-            nostr.innerHTML = "matches";
-            copass.style.borderColor = "#356212";
-            nmsg.style.color = "#356212";
-        }else{
-            nostr.innerHTML = "does not match";
-            copass.style.borderColor = "#b22323";
-            nmsg.style.color = "#b22323";
-        }
-    };
+email.addEventListener('input',() =>{
+    validateEmail();
+});
 
-    function validateEmail() {
-        if (!email.value.match(/^[A-Za-z0-9\._\-]*[@][A-Za-z0-9\.]*[a-z0-9]{2,4}$/)) {
-            estr.innerHTML = "Invalid email structure";
-            email.style.borderColor = "#bc0606";
-            emsg.style.color = "#bc0606";
-            return false;
-        } else {
-            estr.innerHTML = "Valid email";
-            email.style.borderColor = "#356212";
-            emsg.style.color = "#356212";
-            return true;
-        }
+function checkPasswordConfirmation() {
+    if(copass.value.length > 0){
+        nmsg.style.display = "block";
+    }else{
+        nmsg.style.display = "none";
+        copass.style.backgroundColor = "rgba(255,255,255,0.2)";
+        return;
+    }
+    if(copass.value === pass.value ){
+        nostr.innerHTML = "matches";
+        copass.style.backgroundColor = "rgba(81, 182, 14, 0.651)";
+        nmsg.style.color = "#5bec64";
+    }else{
+        nostr.innerHTML = "does not match";
+        copass.style.backgroundColor = "rgba(211, 12, 12, 0.486)";
+        nmsg.style.color = "#b22323";
+    }
+};
+
+function validateEmail() {
+    if (!email.value.match(/^[A-Za-z0-9\._\-]*[@][A-Za-z0-9\.]*[a-z0-9]{2,4}$/)) {
+        estr.innerHTML = "Invalid email structure";
+        email.style.borderColor = "#bc0606";
+        emsg.style.color = "#bc0606";
+        return false;
+    } else {
+        estr.innerHTML = "Valid email";
+        email.style.borderColor = "#5bec64";
+        emsg.style.color = "#5bec64";
+        return true;

```

Comment:
same problem in this part, maybe the top, margin are okay, but if you make classes then you could re-use this class in more context when validating other forms not just here